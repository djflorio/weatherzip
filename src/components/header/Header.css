.header {
  height: 170px;
}

.header__sun {
  width: 120px;
  animation: bounceIn 1s;
}

.header__sun-container {
  position: absolute;
  top: 10px;
  left: 40px;
  animation: rotate 13s infinite linear;
}

.header__cloud {
  position: absolute;
}

.header__cloud--1 {
  top: 60px;
  left: 10px;
  animation: bounceIn 1s 0.1s both;
}

.header__cloud--2 {
  top: 60px;
  left: 80px;
  animation: bounceIn 1s 0.3s both;
}

.header__title {
  position: absolute;
  top: 50px;
  left: 90px;
  font-family: 'Fredoka One', cursive;
  font-size: 3em;
  color: #efefef;
  text-shadow: 0 0 2px #111;
  margin: 0;
  animation: bounceInLeft 1s 0.3s both;
}

@media (max-width: 450px) {
  .header__title {
    font-size: 2.3em;
  }
}