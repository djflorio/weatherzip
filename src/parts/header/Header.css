.header__sun {
  position: absolute;
  top: 10px;
  left: 40px;
  width: 120px;
  animation: rotate 13s infinite linear;
}

.header__cloud {
  position: absolute;
}

.header__cloud--1 {
  top: 60px;
  left: 10px;
}

.header__cloud--2 {
  top: 60px;
  left: 80px;
}

.header__title {
  position: absolute;
  top: 50px;
  left: 90px;
  font-family: 'Fredoka One', cursive;
  font-size: 3em;
  color: #efefef;
  text-shadow: 0 0 2px #111;
  margin: 0;
}