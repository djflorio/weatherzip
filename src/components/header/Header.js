// Third party
import React from 'react';

// Assets
import './Header.css';
import sun from '../../assets/img/sun.svg';
import cloud1 from '../../assets/img/cloud1.svg';
import cloud2 from '../../assets/img/cloud2.svg';


const Header = () => (
  <div className="header">
    <span className="header__sun-container">
      <img className="header__sun" src={sun} />
    </span>
    <img className="header__cloud header__cloud--1" src={cloud1} />
    <img className="header__cloud header__cloud--2" src={cloud2} />
    <h1 className="header__title">
      WeatherZip
    </h1>
  </div>
)

export default Header;