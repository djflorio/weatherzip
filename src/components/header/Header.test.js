import Header from './Header';

it("renders correctly", () => {
  const wrapper = shallow(
    <Header />
  );

  expect(wrapper).toMatchSnapshot();
});