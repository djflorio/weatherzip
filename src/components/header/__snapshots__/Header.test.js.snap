// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="header"
>
  <span
    className="header__sun-container"
  >
    <img
      alt="sun"
      className="header__sun"
      src="sun.svg"
    />
  </span>
  <img
    alt="cloud"
    className="header__cloud header__cloud--1"
    src="cloud1.svg"
  />
  <img
    alt="cloud"
    className="header__cloud header__cloud--2"
    src="cloud2.svg"
  />
  <h1
    className="header__title"
  >
    WeatherZip
  </h1>
</div>
`;
